### ISSUE TITLE
Terraform fails to import a CloudTrail with 'Organization Trail' set to true

### ISSUE THREAD

**Issue by navi7** [terraform-providers/terraform-provider-aws#6561]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.10
+ provider.aws v1.46.0

### Affected Resource(s)

* aws_cloudtrail

### Terraform Configuration Files

```hcl
resource "aws_cloudtrail" "organizations" {
  name           = "trail-organizations"
  s3_bucket_name = "the-bucket"
  s3_key_prefix  = "organizations"
}
```

### Debug Output

https://gist.github.com/navi7/0aafa5f3a057d4319677cdcfeb19eba9
### Panic Output

None

### Expected Behavior

Trail imported

### Actual Behavior

Error reported

### Steps to Reproduce

* manually create the CloudTrail trail with name `trail-organizations`
* set `Apply trail to my organization` to *yes*
* select/create an S3 bucket

**Comment by gwvandesteeg** [comment]
Be nice to also be able to create cloud trails that are organisation wide
As per AWS CLI, it'll be another Boolean Attribute to add to the cloudtrail.
- aws cloudtrail create-trail --is-organization-trail

**Comment by bflad** [comment]
Hi @navi7 👋 Sorry you are running into trouble.

Are you able to verify that your Terraform AWS provider is pointing to the CloudTrail trail home region (in your case `us-east-2`)? You have a few options to verify:

* Checking the Terraform [debug logging](https://www.terraform.io/docs/internals/debugging.html), e.g. `TF_LOG=debug terraform import aws_cloudtrail.organizations trail-organizations`
* Temporarily adding an output with the provider region name, e.g.

```hcl
data "aws_region" "current" {}

output "aws_region" {
  value = "${data.aws_region.current.name}"
}
```

I don't believe there is any specific logic that should preventing the Organizations enabled trail from working with the Terraform `aws_cloudtrail` resource (even though the resource doesn't currently support enabling Organizations support itself yet), its just calling `DescribeTrails` with the name:

https://github.com/terraform-providers/terraform-provider-aws/blob/9e64e8062785e7add88c0ea27cfb15da0ce813bc/aws/resource_aws_cloudtrail.go#L212-L235

**Comment by bflad** [comment]
@gwvandesteeg I have created a separate feature request issue for implementing the CloudTrail Organizations trail support in the resource (if my hunch is correct above, should be separate from this issue): #6579

**Comment by navi7** [comment]
@bflad Thanks for taking a look. Indeed, there is something 'fishy' with the regions used when one wants to set up a CloudTrail for organizations.

Namely, this document https://docs.aws.amazon.com/organizations/latest/userguide/orgs_tutorials_cwe.html#tutorial-cwe-step1 explicitly states:
> On the navigation bar in the upper-right corner of the console, choose the US East (N. Virginia) Region. If you choose a different region, AWS Organizations doesn't appear as an option in the CloudWatch Events configuration settings, and CloudTrail doesn't capture information about AWS Organizations.

Since I have AWS provider region set to `eu-west-3`, it seems that tripped it up. I have now created the trail in `eu-west-3` region (Console allowed me to do that despite what the doc says) and I have imported  the trail successfully.
